bin/check-node-version

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 const requiredVersion = require( '../package' ).engines.node,
-	semver = require( process.env.SEMVER_GLOBAL_PATH );
+	semver = require( 'semver' );
```

Comment:
The reason this was done was to check the node version before the NPM install was run.